server/lib/src/constants/acp.rs

```diff
@@ -1291,7 +1291,14 @@ lazy_static! {
         ("acp_search_attr", Value::new_iutf8("description")),
         ("acp_search_attr", Value::new_iutf8("badlist_password")),
         ("acp_modify_removedattr", Value::new_iutf8("badlist_password")),
-        ("acp_modify_presentattr", Value::new_iutf8("badlist_password"))
+        ("acp_modify_presentattr", Value::new_iutf8("badlist_password")),
```

Comment:
Should probably be a seperate ACP for IDM_ACP_SYSTEM_CONFIG_SESSION_EXP, and then we tie it to the group member instead? That matches in with the other future changes to fine-grain acp later.

Reply:
sure!